backend/jottacloud/jottacloud.go

```diff
@@ -61,27 +61,25 @@ const (
 	defaultTokenURL = "https://id.jottacloud.com/auth/realms/jottacloud/protocol/openid-connect/token"
 	defaultClientID = "jottacli"
 
+	whitelabelClientID = "desktop"
+
+	whitelabelTeliaSeDomain = "cloud-auth.telia.se"
```

Comment:
These could probably be a `map[tag]struct{ domain, realm string}` couldn't they?

That would simplify the code a bit below.

You could also build the `[]fs.OptionExample` passed to the `fs.ConfigChooseExclusiveFixed("whitelabel_type_done", ...` function programatically from this struct. Might need to have a name in for that too.